css-view-transitions-2/Overview.bs

```diff
@@ -304,22 +308,26 @@ The <dfn attribute for=PageRevealEvent>viewTransition</dfn> [=getter steps=] are
 	<div algorithm="monkey patch to reactivation">
 		Run the following step in [=Document/reactivate=], before step 4 (querying for [=page showing=]):
 
-		1. If |document| is not [=render-blocked=], then [=reveal document|reveal=] |document|.
+		1. [=Reveal document|reveal=] |document|.
 	</div>
 
 	<div algorithm="page reveal">
 	To <dfn>reveal {{Document}}</dfn> |document|:
 		1. Assert: |document|'s [=page showing=] is false.
 
-		1. Fire a new event named <code>reveal</code> on |document|'s [=relevant global object=],
-			using {{PageRevealEvent}}.
+		1. If |document| is [=render-blocked=], then return.
 
 		1. Let |transition| be the result of getting the [=inbound cross-document view-transition=] for |document|.
 
+		1. If |document| does not [=opt in to cross-document view transitions=], then [=skip the view transition|skip=] |transition| and set |transition| to null.
```

Comment:
nit: Do we need a null check for `transition` here? Also, I think you want say set document's active view transition to null. Mapping this to code, `transition` is now a local variable in this algorithm so resetting it won't reset the member variable on Document.

Reply:
skip transition already sets the active view transition to null.
Fixed the nit.